packages/rich-text-types/src/blocks.ts

```diff
@@ -21,6 +21,7 @@ export enum BLOCKS {
 
   EMBEDDED_ENTRY = 'embedded-entry-block',
   EMBEDDED_ASSET = 'embedded-asset-block',
+  EMBEDDED_RESOURCE = 'embedded-resource-block',
```

Comment:
I actually wonder about this name. it's very generic, but the type interfaces being defined that represent it are very Contentful:Entry specific. This feels like enough of a future-impacting risk that we should address this now before this lands.

Reply:
For the value: `embedded-resource-block` follows the names proposed in the Rich Text CEP (`embedded-resource-block`, `embedded-resource-inline` and `resource-hyperlink`). If you mean the key here, I'm open to renaming suggestions.